translator/conventions/opencensus.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 OpenTelemetry Authors
+// Copyright 2019 OpenTelemetry Authors
```

Comment:
Remove the year?

Reply:
I'll set to 2020 to match the other files. We can remove from everywhere once there is a decision.